content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -938,6 +938,17 @@ creates Pods with the finalizer `batch.kubernetes.io/job-tracking`. The
 controller removes the finalizer only after the Pod has been accounted for in
 the Job status, allowing the Pod to be removed by other controllers or users.
 
+#### Troubleshooting
+
+- **Pods stuck in Terminating state and Jobs don't complete**
+
+  Check for validating or mutating webhooks for Pod updates. Misconfigured
+  or faulty webhooks may attempt to modify running Pods or deny an update for a
+  Pod that is already running or finished. This causes kube-apiserver to reject
+  the update to remove the Pod finalizers, preventing the job controller from
+  making progress.
+  You should disable and/or fix the Pod update webhooks.
```

Comment:
It helps readers when concept explanations are as short as we can reasonably make them.

I'd prefer to have the actual advice somewhere in https://kubernetes.io/docs/tasks/debug/debug-application/ and have _this_ page hyperlink to the troubleshooting page.

Reply:
uhm... it doesn't fit in the existing articles. We would need a new "Debugging Jobs" or "Debugging control plane", which I don't have the time to write now :(

Reply:
I could put it in "Debugging Pods" as "My pod is stuck in Terminating"

Reply:
@alculquicondor we can file an issue asking for another (docs) contributor to pick up the baton here. Would that work?

Reply:
I'm also OK with adding a section to Debugging Pods as an interim thing, and making the Debugging Jobs task page a longer term ambition.

Reply:
I put it in "Debugging Pods". It might be the most appropriate place long term, after all.